### PR TITLE
Account deletion request v2

### PR DESCRIPTION
**Pull request by charles-cowart**
Supersedes https://github.com/biocore/microsetta-interface/pull/225

### CODE REVIEW

**Comment by charles-cowart** [comment]
@cassidysymons All relevant changes migrated from the old PR. Haven't tested this locally yet. Will do so later tonight.

**Comment by charles-cowart** [comment]
Created a new microsetta_private_api environment using master-overhaul branch and recreated errors seen in CI. Currently debugging each in turn. Will keep you posted.

**Comment by charles-cowart** [comment]
@cassidysymons This fix requires one small fix to microsetta-private-api as well. Putting together a PR for that now.

**Comment by charles-cowart** [comment]
test_existing_user_to_secondary_survey() is failing in CI but not locally. Comparing local and CI environments for any differences.

**Comment by charles-cowart** [comment]
@cassidysymons I can't recreate the test_existing_user_to_secondary_survey() failure locally after reviewing the CI configuration, and using both master-overhaul and master branches of private-api. I've added two lines of code to return the entire result of the API call so I can determine what is being sent. If you don't mind, can you please rerun the integration test in CI so I can capture the result? Thanks!

**Comment by charles-cowart** [comment]
@cassidysymons Thanks! I looked at the run and it doesn't look like the full result printed out, even though it does locally and even though self.maxDiff = None. I'll think of something else.

**Comment by charles-cowart** [comment]
@cassidysymons It took some doing, but I was able to run and debug the Github workflow locally using https://github.com/nektos/act. I did have to modify the workflow significantly as act doesn't support services, and forced microsetta-private-api to pull master-overhaul branch, but I was able to confirm that the result included consent text. I believe this should pass all tests as long as master-overhaul branch of private-api is used.

**Comment by cassidysymons** [comment]
@charles-cowart Thanks for getting that resolved! I'll give the PR a once-over in the morning and make sure it doesn't need anything else.

File: microsetta_interface/templates/account_details.jinja2 [hunk 1]
```diff
@@ -406,4 +410,37 @@
...
 </div>
+{% if not admin_mode %}
+    {% if requested_deletion %}
+        <div class="container">
+            <div class="row">
+                <div class="col-12 {% if samples|length > 0 %}tooltipper{% endif %}" data-title="{{ _('Request account deletion.') }}">
+                    <p>
+                    </p>
+                        <center>
+                            Your account removal request is being reviewed. You will be notified via email once your account has been deleted.
```
Comment (wasade) [inline microsetta_interface/templates/account_details.jinja2:421]: Can this be placed within a gettext call? (e.g., `_("...")` )

File: microsetta_interface/templates/account_details.jinja2 [hunk 2]
```diff
@@ -406,4 +410,37 @@
...
+                <div class="col-12 {% if samples|length > 0 %}tooltipper{% endif %}" data-title="{{ _('Request account deletion.') }}">
+                    <p>
+                    </p>
+                        <center>
+                            <form name="request_delete_{{account.account_id}}_form"
+                             method="post" action="/accounts/{{ account.account_id }}/request/remove"
+                             onsubmit="return verifyDeleteUserRequest();">
+                                  {{ _('If you wish to delete this account, please click the following button to submit your request to an administrator.') }}
+                            <button type="submit" class="btn btn-outline-danger">{{ _('Request Account Deletion') }}</button>
+                            <br>IMPORTANT: Once you click this button, the request cannot be undone. Your account cannot be restored after it has been deleted.</br>
```
Comment (wasade) [inline microsetta_interface/templates/account_details.jinja2:438]: Same here on gettext

File: microsetta_interface/templates/request_account_deletion_confirm.jinja2 [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+</style>
+{% endblock %}
+
+{% block content %}
+<center>
+    <div class="card"  style="max-width: 800px; text-align: left;">
+        <h5 class="card-header" style="text-align: center">{{ _('Account Removal Requested') }}</h5>
+        <div class="card-body">
+            <center>
+                {{ _('We are sorry to see you go! Your request has been logged and you will receive an email notification once your account has been deleted.') }}
```
Comment (wasade) [inline microsetta_interface/templates/request_account_deletion_confirm.jinja2:27]: "We are sorry to see you go! Your request has been logged, and an administrator will review the request shortly. You will receive an email notification once your account has been deleted."

File: microsetta_interface/templates/source.jinja2 [hunk 4]
```diff
@@ -271,16 +271,16 @@
...
                 </div>
-                <div class="col-2" style="text-align: right">
+                <div class="col">
                     {% if detail.new_tab %}
                     <a class="btn btn-primary" href="/accounts/{{ account_id }}/sources/{{ source_id }}/take_survey?survey_template_id={{ detail.survey_template_id }}" target="_blank">
                     {% else %}
                     <a class="btn btn-primary" href="/accounts/{{ account_id }}/sources/{{ source_id }}/take_survey?survey_template_id={{ detail.survey_template_id }}">
                     {% endif %}
-                        {{ _('Start') }}
+                        Start
```
Comment (wasade) [inline microsetta_interface/templates/source.jinja2:283]: Please revert, this needs to be in gettext

**Pull request closed without merging**